intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md

```diff
@@ -15,7 +15,7 @@ head from 1/16 scale feature map has nine clustered prior boxes.
 
 | Metric                                                        | Value                   |
 |---------------------------------------------------------------|-------------------------|
-| AP ([WIDER](http://shuoyang1213.me/WIDERFACE/)                | 83.00%                  |
+| AP ([WIDER])(http://shuoyang1213.me/WIDERFACE/)               | 83.00%                  |
```

Comment:
`AP [WIDER](http://shuoyang1213.me/WIDERFACE/)` - this is correct change

Reply:
or `([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/))`

Reply:
This one should be the correct one
([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/))

Reply:
fixed